packages/i18n-utils/src/utils.ts

```diff
@@ -226,3 +226,26 @@ export function isMagnificentLocale( locale: string ): boolean {
 export function isTranslatedIncompletely( locale: string ) {
 	return getLanguage( locale )?.isTranslatedIncompletely === true;
 }
+
+/**
+ * Removes the Locale from the given path and returns the real path.
+ *
+ * @param {string}  path  Original path
+ * @returns {string} path The real path (with locale removed)
+ */
+export function getRealPathName( path: string ) {
```

Comment:
Also, we seem to have `removeLocaleFromPath` that should do a similar job. Have you had the chance to explore using that instead of introducing a new function? 

https://github.com/Automattic/wp-calypso/blob/0d17396107f113bd675fb37ff0e0559817975045/packages/i18n-utils/src/utils.ts#L173

Reply:
@tyxla - thanks for pointing me to this. So it looks like `removeLocaleFromPath` only accepts paths like /my-path/es - however for themes / plugins we have the locale in front, like: /es/themes - so I introduced a new function `removeLocaleFromPathLocaleInFront` similar to `removeLocaleFromPath` and got rid of the `getRealPath` method.

Reply:
Ah, that sounds like a consistency mess 😅 Not much we can do to improve consistency at this point, so I'd recommend making sure that the intents are clear as to whether we'd like to handle locales in front, in the end, or both.